source/_patterns/_variables.scss

```diff
@@ -51,8 +51,16 @@ $font-system: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
 // => for IE11 support: NEVER use font: size/line-height $font-system; (short form) but font-family: $font-system instead!
 
 /*** SHADOWS ***/
-$shadow-button-default: 0 1px 2px 0 rgba(0, 0, 0, .24), 0 0 2px 0 rgba(0, 0, 0, .12);
-$shadow-button-hover:   0 4px 8px 0 rgba(0, 0, 0, .24), 0 0 8px 0 rgba(0, 0, 0, .12);
+$shadow-default: 0 1px 2px 0 rgba(0, 0, 0, .24), 0 0 2px 0 rgba(0, 0, 0, .12);
+$shadow-hover:   0 4px 8px 0 rgba(0, 0, 0, .24), 0 0 8px 0 rgba(0, 0, 0, .12);
+
+/*** BREAKPOINT AND CONTENT related variables ***/
+$smartphonesize: 320px;
```

Comment:
Grundsätzlich, sehr cool, dass es jetzt in variablen ausgelagert ist. 
Ich wäre jedoch für:
$smartphone-size
$tablet-size
$desktop-size
$desktop-wide-size

Dann ist das auch konform zu den mixin namen und code standard.

Reply:
Schon drin: 
https://github.com/mmz-srf/srf-frontend-framework/blob/master/source/_patterns/_breakpoints.scss

Reply:
@pchr-srf finde meine Lösung («hands down») eleganter ;-)

Reply:
Dann viel Spass beim Mergen 👍